### ISSUE TITLE
Add a shortcut from colour picker to global styles

### ISSUE THREAD

**Issue by jameskoster** [WordPress/gutenberg#41871]
When styling a block in the Site Editor, it's feasible that you might want to go and edit your global colour palettes directly from the block. It could be a nice enhancement to connect the colour picker to the Colour panel in global styles.

There exists already a pattern we might use to achieve this:

<img width="281" alt="Screenshot 2022-06-22 at 12 53 55" src="https://user-images.githubusercontent.com/846565/175022952-913e2ce5-b178-4779-be93-0374d058ee8a.png">
<img width="397" alt="Screenshot 2022-06-22 at 12 54 05" src="https://user-images.githubusercontent.com/846565/175022959-971ec4aa-bd55-40f5-9a33-d2e6d4c036de.png">

We could potentially reuse that here:

<img width="722" alt="Screenshot 2022-06-22 at 12 53 31" src="https://user-images.githubusercontent.com/846565/175022860-fda39475-39ac-4de6-9c70-58520a492bae.png">

**Comment by mtias** [comment]
I think this might be a bit too intense for a rare action.

**Comment by jameskoster** [comment]
Yup there are other options to explore in the Color panel itself. Text link, ellipsis menu, etc.

**Comment by karmatosed** [comment]
I am not sure this is valid anymore. We have this today:

<img width="726" alt="Image" src="https://github.com/user-attachments/assets/aa5da857-57a5-48ec-a585-34f914cd7203" />

For now, I am going to close this, but we can always reopen for further consideration.
